Library/Homebrew/extend/string.rb

```diff
@@ -79,3 +79,36 @@ def get_make_var(flag)
     self[/^#{Regexp.escape(flag)}[ \t]*=[ \t]*(.*)$/, 1]
   end
 end
+
+# Taken from https://github.com/marcandre/backports
```

Comment:
Can we stick this in some sort of `vendor` directory and link directly to https://github.com/marcandre/backports/blob/master/lib/backports/2.4.0/string/match.rb?